apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dice in a cup
+  """
+
+  @spec new(integer) :: [integer]
+  def new(n, acc \\ [])
+  def new(0, acc), do: acc
+  def new(n, acc) do
+    new(n-1, [:rand.uniform(6) | acc])
+  end
```

Comment:
I was actually looking for something like this. There wasn't a `repeat` function or something similar in the `Enum` module.

The only issue is how do I pass in a parameter, namely `6`, to `&:rand.uniform`? Thanks